data/types/base.go

```diff
@@ -62,7 +63,8 @@ type Base struct {
 	Source            *string                                      `json:"source,omitempty" bson:"source,omitempty"`
 	Tags              *[]string                                    `json:"tags,omitempty" bson:"tags,omitempty"`
 	Time              *string                                      `json:"time,omitempty" bson:"time,omitempty"`
-	TimezoneOffset    *int                                         `json:"timezoneOffset,omitempty" bson:"timezoneOffset,omitempty"`
+	TimeZoneName      *string                                      `json:"timezone,omitempty" bson:"timezone,omitempty"`             // TODO: Rename to timeZoneName
```

Comment:
cool - was going to ask that question. Why not `timeZoneName` rather than `timezone` ...

Reply:
Eventually will migrate everything to proper two word usage, but for now it is piecemeal.